mx-rust-semantics/main/modules/address.md

```diff
@@ -28,7 +28,7 @@ module MX-RUST-MODULES-ADDRESS
             ( struct
                 ( #token("ManagedAddress", "Identifier"):Identifier
                 , #token("mx_address_value", "Identifier"):Identifier |-> AddressValueId:Int
-                  _:Map
```

Comment:
Could you elaborate on a scenario where a non-empty map here should prevent the rewriting operation of `rustValueToMx`? This question is more for clarification purposes.

Reply:
This map should always be empty. Forcing an empty map here similar to an assertion - if we ever change the definition of the "ManagedAddress" to include more fields, this rule will fail to apply and we will have to update it.